.github/workflows/contrib-openai.yml

```diff
@@ -4,7 +4,7 @@
 name: OpenAI4ContribTests
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
To run this test, the PR also needs to be made from the main repo and not the forked repo.